src/api/IBMiContent.ts

```diff
@@ -187,6 +188,10 @@ export default class IBMiContent {
         });
 
         if (copyResult.code === 0) {
+          const messages = Tools.parseMessages(copyResult.stderr);
+            if (messages.findId("CPIA083")) {
+              window.showWarningMessage( `${library}/${sourceFile}(${member}) was saved with truncated records!`);
+            }
```

Comment:
Watch out for the indentation 😉
```suggestion
          const messages = Tools.parseMessages(copyResult.stderr);
          if (messages.findId("CPIA083")) {
            window.showWarningMessage( `${library}/${sourceFile}(${member}) was saved with truncated records!`);
          }
```